apps/web/lib/services/client.service.ts

```
@@ -0,0 +1,141 @@
+    } catch (error) {
+      console.error("Error fetching clients:", error);
+      throw error;
+    }
+  }
+
+  /**
+   * Get all clients in ClientInfo format
+   * @returns Array of clients in ClientInfo format
+   */
+  static async getAllClientInfos(): Promise<Client[]> {
+    const dbClients = await this.getAllClients();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)